docs/releases.rst

```diff
@@ -1,6 +1,333 @@
 Release notes
 =============
 
+Mailu 2.0 - 2023-04-03
+----------------------
+
+Mailu 2.0 is finally available. It is vital to read the `Upgrading` section before upgrading to Mailu 2.0.
+
+Highlights
+``````````
+
+This is an overview of the major features introduced in Mailu 2.0.
+
+Multi-arch images (ARM support)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Mailu project now ships multi-arch images for the architectures:
+
+- linux/amd64
+- linux/arm64/v8
+- linux/arm/v7
+
+It is now possible to run Mailu on most ARM hardware such as the Raspberry Pi.
+
+Auto-configuration for client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the domain details page, there are new DNS records for enabling DNS auto-client configuration.
+Provided they are configured, email clients will make use of them to auto-configure.
+
+If a reverse proxy is in use, settings might have to be tweaked.
+
+For Apple users, the client setup page now offers an autoconfiguration link to automatically configure
+their device.
+
+RESTFul API
+^^^^^^^^^^^
+
+Mailu offers a RESTful API for changing the Mailu configuration.
+Now, anything that can be configured via the Mailu web administration interface
+can also be configured via the Mailu RESTful API.
+
+Configuring a new domain or add new users can be fully automated now.
+
+The current API makes use of a single API token for authentication.
+In a future release this will likely be re-visited.
+
+For more information refer to the `Mailu RESTful API` page.
+
+Header authentication support (use external identity providers)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is now possible to use different authentication systems (such as keycloak, authentik, vouch-proxy) to handle the authentication of Mailu users.
+This can be used to enable Single Sign On from other IDentity Providers via protocols such as OIDC or SAML2.
+
+For more information see `Header authentication using an external proxy` in the configuration reference.
+
+Better anti-spoofing protection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously Mailu would reject emails where an attacker spoofs the envelope-From. Now Mailu also checks the header-From for any hosted domain.
+It won't let any email which pretends to be for any of the local domains through unless they pass DMARC.
```

Comment:
Does this mean DMARC setup is mandatory to use Mailu? Maybe that should be clarified then.

Reply:
Only if you intend to send emails for a domain from somewhere that isn't the Mailu instance.

Reply:
83ce32f7